content/en/continuous_integration/setup_tests/java.md

```diff
@@ -12,36 +12,25 @@ further_reading:
 
 ## Compatibility
 
-Supported Test frameworks:
-* JUnit 4.10+
-* JUnit 5.3+
-* TestNG 6.4+
-* Frameworks that are based on JUnit, such as Spock Framework and Cucumber-Junit
-
-Supported CI providers:
-* Appveyor
-* Azure Pipelines
-* BitBucket
-* BuildKite
-* CircleCI
-* GitHub Actions
-* GitLab
-* Jenkins
-* TravisCI
+Supported test frameworks:
+* JUnit >= 4.10 and >= 5.3
+  * Also includes any test framework based on JUnit, such as Spock Framework and Cucumber-Junit
+* TestNG >= 6.4
 
 ## Prerequisites
 
 [Install the Datadog Agent to collect tests data][1].
 
 ## Installing the Java tracer
 
-### Maven
+{{< tabs >}}
+{{% tab "Maven" %}}
 
-Add a new Maven profile in your root `pom.xml` configuring the Datadog Java tracer dependency and the `javaagent` arg property, replacing `$VERSION` with the latest version of the tracer accessible from the [Maven Repository][2]: 
+Add a new Maven profile in your root `pom.xml` configuring the Datadog Java tracer dependency and the `javaagent` arg property, replacing `$VERSION` with the latest version of the tracer accessible from the [Maven Repository][2] (without the preceding `v`): ![Maven Central][7]
```

Comment:
⚠️  this is not rendering correctly

Reply:
Link rendering seems to be broken in general for this page

Reply:
Good catch - it might have been when adding the tabs, fixing.

Reply:
@edrevo should be fixed now